File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/Chart/GrandSlam.cs
namespace Kendo.Mvc.Examples.Models
{
    public class GrandSlam
    {
        public GrandSlam()
        {
        }

        public GrandSlam(int year, int win, int loss, string extremum = null)
        {
            Year = year;
            Win = win;
            Loss = loss;
            Extremum = extremum;
        }

        public int Year { get; set; }
        public int Win { get; set; }
        public int Loss { get; set; }
        public string Extremum { get; set; }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/GrandSlam.java
package com.kendoui.spring.models;

public class GrandSlam {
    private int year;
    private int win;
    private int loss;
    private String extremum;
    
    public GrandSlam(int year, int win, int loss, String extremum) {
        setYear(year);
        setWin(win);
        setLoss(loss);
        setExtremum(extremum);
    }
    
    public int getYear() {
        return year;
    }
    
    public void setYear(int year) {
        this.year = year;
    }

    public int getWin() {
        return win;
    }

    public void setWin(int win) {
        this.win = win;
    }

    public int getLoss() {
        return loss;
    }

    public void setLoss(int loss) {
        this.loss = loss;
    }

    public String getExtremum() {
        return extremum;
    }

    public void setExtremum(String extremum) {
        this.extremum = extremum;
    }
}
